src/common/OneDSLoggerTelemetry/oneDSLogger.ts

```diff
@@ -342,8 +342,15 @@ export class OneDSLogger implements ITelemetryLogger{
 
     // eslint-disable-next-line @typescript-eslint/no-explicit-any
     private populateVscodeDesktopAttributes(envelope: any){
-        // TODO: this method helps in populating desktop attributes.
-        console.log(envelope);
+        if (envelope.data.eventName == desktopExtTelemetryEventNames.DESKTOP_EXTENSION_INIT_CONTEXT) {
+            OneDSLogger.contextInfo.orgId = JSON.parse(envelope.data.eventInfo).OrgId;
+            OneDSLogger.contextInfo.envId = JSON.parse(envelope.data.eventInfo).EnvironmentId;
+            // TODO: Populate website id
+            OneDSLogger.contextInfo.websiteId = 'test'
+        }
+        envelope.data.measurements = envelope.data.measurement;
+        envelope.data.timestamp = envelope.time;
+        envelope.data.sdkVersion = envelope.ext.sdk.ver;
```

Comment:
Why this is needed?

Reply:
These are phase 1 attributes - therefore added here for desktop extension to get them populated.

Reply:
They are auto populated.

Reply:
@BidishaMS As per my analysis, they are not getting auto-populated. Reason being - they are not part of envelope.data that is being received. That's why different mapping was needed.

Reply:
Discussed on call, removing sdk version. This is getting populated in envelope.ext.sdk.ver & will be populated at the backend.